Jellyfin.Plugin.SubtitleExtract/Tools/SubtitleExtractor.cs

```diff
@@ -42,28 +42,30 @@ public async Task Run(BaseItem video, CancellationToken cancellationToken)
             try
             {
                 var mediaSourceId = video.Id.ToString("N", CultureInfo.InvariantCulture);
-                var subtitleMediaStream = video
+                var extractableSubtitleMediaStream = video
                     .GetMediaStreams()
-                    .FirstOrDefault(stream => stream is { IsTextSubtitleStream: true, SupportsExternalStream: true, IsExternal: false });
-                if (subtitleMediaStream is not null)
+                    .First(s => s is { IsExtractableSubtitleStream: true, SupportsExternalStream: true });
```

Comment:
This should still be `FirstOrDefault` as there is no guarantee that a video has an ex tractable subtitle stream